[DOCS] Whats new [8.1] (#1680)
elastic/security-docs: docs/whats-new.asciidoc

[[whats-new]]
[chapter]
= What's new in 8.1

Here are the highlights of what’s new and improved in {elastic-sec}!

For detailed information about this release, check out the <<release-notes, Release notes>>.

Other versions: {security-guide-all}/8.0/whats-new.html[8.0] | {security-guide-all}/7.17/whats-new.html[7.17] | {security-guide-all}/7.16/whats-new.html[7.16] | {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
{security-guide-all}/7.9/whats-new.html[7.9]

// NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
// tag::notable-highlights[]

[discrete]
[[name-changes-8.1]]
== Terminology changes

The *Exceptions* page is now named *Exception lists*.

[discrete]
[[features-8.1]]
== New features

*Rules page enhancements*

The following applies to the *Rules* and *Rules monitoring* tabs.

> As an optional technical preview, you can now {security-guide}/alerts-ui-monitor.html[sort all columns]. To enable the technical preview, switch the *Technical preview* toggle on. If you experience performance issues, you can turn this setting off.

Reviewer: ```suggestion
As an optional technical preview, you can now {security-guide}/alerts-ui-monitor.html[sort these pages by any column]. To enable the technical preview, switch the *Technical preview* toggle on. If you experience performance issues, you can turn this setting off.
```

[role="screenshot"]
image::whats-new/images/8.1/column-sort.gif[]

The following {security-guide}/rules-ui-management.html[new bulk actions] can be performed on multiple rules simultaneously:

* Add index patterns
* Delete index patterns
* Add tags
* Delete tags

Select the *Bulk actions* menu to view all available bulk options.

[role="screenshot"]
image::whats-new/images/8.1/bulk-actions.png[]

*Event filters and host isolation exceptions can be applied per policy*

> Users with a https://www.elastic.co/pricing/[Platinum or Enterprise license] can apply new or existing {security-guide}/event-filters.html[event filters] and {security-guide}/host-isolation-exceptions.html[host isolation exceptions] to a specific integration policy instead of globally. This allows you to configure settings that may be unique to a specific group within your network.

Reviewer: ```suggestion
Users with a https://www.elastic.co/pricing/[Platinum or Enterprise license] can now apply new or existing {security-guide}/event-filters.html[event filters] and {security-guide}/host-isolation-exceptions.html[host isolation exceptions] to a specific integration policy instead of globally. This allows you to configure settings that may be unique to a specific group within your network.
```

*Network map data requires new permissions*

> To view network map data, you need at least `Read` privileges for `Maps`. Refer to {security-guide}/conf-map-ui.html[Configure network map data] for more information.

Reviewer: ```suggestion
To view network map data, you now need at least `Read` privileges for `Maps`. Refer to {security-guide}/conf-map-ui.html[Configure network map data] for more information.
```

*Alert enhancements*

On the Alerts page, the Trend and Count tables now allow you to aggregate alert data based on any aggregatable ECS field. Previously, only a handful of fields were visible.

[role="screenshot"]
image::whats-new/images/8.1/counts-table.gif[]

> The {security-guide}/alerts-ui-manage.html#view-alert-details[alert details flyout] has a new layout that highlights relevant fields. You can also now update the alert's status directly from the flyout.

Reviewer: ```suggestion
The {security-guide}/alerts-ui-manage.html#view-alert-details[alert details flyout] has a new layout that highlights relevant fields. You can also now update an alert's status directly from the flyout.
```

[role="screenshot"]
image::whats-new/images/8.1/column-sort.gif[]

The following {security-guide}/rules-ui-management.html[new actions] can be performed on multiple rules simultaneously:

* Add index patterns
* Delete index patterns
* Add tags
* Delete tags

Select the *Bulk actions* menu to view all available bulk options.

[role="screenshot"]
image::whats-new/images/8.1/bulk-actions.png[]

*Event filters and host isolation exceptions can be applied per policy*

You can apply new or existing {security-guide}/event-filters.html[event filters] and {security-guide}/host-isolation-exceptions.html[host isolation exceptions] to a specific policy instead of globally. This allows you to configure settings that may be unique to a specific group within your network.

*Network map data requires new permissions*

To view network map data, you need at least `Read` privileges for `Maps`. Refer to {security-guide}/conf-map-ui.html[Configure network map data] for more information.

*Alert enhancements*

> On the Alerts page, the Trend and Count tables on the  now allow you to aggregate alert data based on any aggregatable ECS field. Previously, only a handful of fields were visible.

Reviewer 2: ```suggestion
On the Alerts page, the Trend and Count tables on the  now allow you to aggregate alert data based on any aggregatable ECS field. Previously, only a handful of fields were available.
```
